client/components/AttributionEditor/AttributionDetailControls.tsx

```diff
@@ -105,28 +121,41 @@ const AttributionDetailControls = (props: Props) => {
 						}
 					}}
 					onBlur={(evt) =>
-						onAttributionUpdate({
+						onAttributionUpdateWithValidation({
 							id,
 							affiliation: evt.target.value.trim(),
 						})
 					}
 				/>
 				{isShadowAttribution && (
-					<InputGroup
+					<InputField
```

Comment:
I needed to use `InputField` bc `InputGroup` does not have the `onError` thingy.